docs/writing-stories/parameters.md

```diff
@@ -56,7 +56,7 @@ export default {
 We can also set the parameters for **all stories** via the `parameters` export of your [`.storybook/preview.js`](../configure/overview.md#configure-story-rendering) file (this is the file where you configure all stories):
 
 ```js
-// Button.story.js
+// preview.js
```

Comment:
```suggestion
// .storybook/preview.js
```

Reply:
users can also have custom storybook config directory.. but I guess the most common case will be `.storybook/preview.js` and whoever has a custom config folder will figure out easily, right?